checker/manual/introduction.tex

```diff
@@ -677,7 +677,7 @@ \subsection{Summary of command-line options\label{checker-options}}
 \item
 \<-AoutputArgsToFile>
 Output the compiler command-line arguments to a file.  Useful when this is not
```

Comment:
Clarify what "this" is referring to.  Also, why is it useful?  (For debugging and reproducibility, I guess.)